app/models/change_note.rb

```diff
@@ -0,0 +1,30 @@
+class ChangeNote < ActiveRecord::Base
+  belongs_to :content_item
+
+  def self.create_from_content_item(payload, content_item)
+    return unless content_item.update_type == "major"
+    create_from_top_level_change_note(payload, content_item) ||
+      create_from_details_hash_change_note(content_item) ||
+      create_from_details_hash_change_history(content_item)
+  end
+
+  def self.create_from_top_level_change_note(payload, content_item)
```

Comment:
Should these class methods be private? Which kinda makes me think that `self.create_from_content_item` shouldn't be a class method, maybe some other object, which would help with passing the payload and content_item around. What do you think?